vocabularies/Org.OData.Core.V1.json

```diff
@@ -512,9 +511,13 @@
         },
         "ContentDispositionType": {
             "$Kind": "ComplexType",
+            "Type": {
+                "$Nullable": true,
```

Comment:
What does absence of this property mean? Is there a default value?

Reply:
I tend towards `DefaultValue="attachment"` because in most cases specifying a `Filename` is done in combination with `attachment`, and would be compatible with the current use of the annotation, only specifying the "download filename".

No default value can also be considered compatible.